addons/event/data/event_demo.xml

```diff
@@ -67,8 +69,47 @@
         <field eval="(DateTime.today()+ timedelta(days=100)).strftime('%Y-%m-%d 20:15:00')" name="date_begin"/>
         <field eval="(DateTime.today()+ timedelta(days=101)).strftime('%Y-%m-%d 00:30:00')" name="date_end"/>
         <field name="event_type_id" ref="event_type_2"/>
-        <field name="address_id" ref="base.res_partner_1"/>
+        <field name="address_id" ref="event.res_partner_location_0"/>
         <field name="stage_id" ref="event_stage_booked"/>
+        <field name="kanban_state">blocked</field>
+    </record>
+
+    <record id="message_event_1_0" model="mail.message">
+        <field name="model">event.event</field>
+        <field name="res_id" ref="event.event_1"/>
+        <field name="body" type="html"><p>Hello Marc Demo,<br/>
+            Our flight authorizations have been revoked due to insurance issues.<br/>
+            Could you take care of it as soon as possible ?</p>
+        </field>
+        <field name="message_type">comment</field>
+        <field name="author_id" ref="base.partner_admin"/>
+    </record>
+    <record id="message_event_1_1" model="mail.message">
+        <field name="model">event.event</field>
+        <field name="res_id" ref="event.event_1"/>
+        <field name="parent_id" ref="message_event_1_0"/>
+        <field name="body" type="html"><p>Hi Mitchell Admin,<br/>I will take care of it today !</p></field>
+        <field name="message_type">comment</field>
```

Comment:
If we want to have full demo data, we can also set subtype (mail.mt_comment for comments, mail.mt_note for note, should probably be sufficient for standard demo).